features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/extensions/publisher/assets/api/themes/default/js/api_subscriptions.js

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+var getAPIUrl;
+var updateSubscription;
+
+$(function () {
+
+    //var TD_API_CONTAINER = '#td-apis';
+
+    //var EV_TD_API = 'eventTDAPISelection';
+
+    //events.register(EV_TD_API);
+
+    // Views.extend('view', {
+    //        id: 'apiListView',
+    //        container: TD_API_CONTAINER,
+    //        partial: 'td-apis-subscriptions',
+    //        beforeRender: function (data) {
+    //     var user = data;
+    //        },
+    //        subscriptions: [EV_TD_API],
+    //        afterRender: function () {}
+    //    });
```

Comment:
Thanks for pointing out this. PR sent for fix https://github.com/wso2/carbon-apimgt/pull/779